yjit/src/codegen.rs

```diff
@@ -5683,6 +5710,40 @@ fn gen_send_iseq(
         argc = lead_num;
     }
 
+    if iseq_has_rest {
+        assert!(argc >= required_num);
+
+        jit_save_pc(jit, asm);
+        gen_save_sp(jit, asm, ctx);
+
+        // We are going to allocate so setting pc and sp.
```

Comment:
```suggestion
        // We are going to allocate so setting pc and sp.
        jit_save_pc(jit, asm);
        gen_save_sp(jit, asm, ctx);

```

It seems more natural to put the comment before doing what's mentioned.